vra7/utils.go

```diff
@@ -97,3 +97,18 @@ func GetConfiguration(componentName string, resourceConfiguration []sdk.Resource
 	}
 	return m
 }
+
+// ConvertToDateTime converts ISO8601 (2020-04-16T00:15:00.700Z) returned from vRA to YYYY-MM-DD HH:MM format
```

Comment:
These functions should not be performing string operations on time strings. Rather the input should be [Parsed](https://golang.org/pkg/time/#Parse) into a time and then use [Format](https://golang.org/pkg/time/#Time.Format) to generate the string.